sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcReadBenchmark.scala

```diff
@@ -320,6 +361,11 @@ object OrcReadBenchmark extends SqlBasedBenchmark {
       columnsBenchmark(1024 * 1024 * 1, 200)
       columnsBenchmark(1024 * 1024 * 1, 300)
     }
+    runBenchmark("Deser Factor Calculation From Wide Columns") {
```

Comment:
Can we add a benchmark for highly compressed scenario like `runBenchmark("Repeated String Scan")`, too?